themes/default/content/challenge/startup-in-a-box.md

```diff
@@ -14,32 +14,37 @@ meta_image: /images/challenge/challenge_cta.png
   <div class="w-full md:w-1/2">
     <h3>Startup in a Box</h3>
     <p class="pr-12">
-      Thinking about turning that side project into a little something more? Follow along to stand up a website for your startup on Amazon S3 with Cloudfront and Checkly, all using Pulumi. When you're done, we'll send you a fancy drink tumbler with a special Pulumipus on it, just for this Challenge!
+      Thinking about turning that side project into a little something more? Follow along to stand up a website for your startup on your favourite cloud provider (AWS or Google Cloud) and Checkly, all using Pulumi. When you're done, we'll send you a fancy drink tumbler with a special Pulumipus on it, just for this Challenge!
```

Comment:
I would actually like the specific resource to be called out. or we can go generic "object storage and CDN (AWS or Google Cloud)"

Reply:
like that idea - will adopt the wording